codegenerator/cli/templates/dynamic/codegen/src/IO.res.hbs

```diff
@@ -167,32 +167,21 @@ let executeBatch = async (
     ~items=inMemoryStore.rawEvents->InMemoryTable.values,
   )
 
-  let setDynamicContracts = executeSet(
-    _,
-    ~dbFunction=DbFunctions.DynamicContractRegistry.batchSet,
-    ~items=inMemoryStore.dynamicContractRegistry->InMemoryTable.values,
-  )
-
-  {{#each entities as | entity |}}
-  let set{{entity.name.capitalized}}s = entityDbExecutionComposer(
-    _,
-    ~entityMod=module(Entities.{{entity.name.capitalized}}),
-    ~inMemoryStore,
-  )
+  let setEntities = Entities.allEntities->Belt.Array.map(entityMod => {
+    entityDbExecutionComposer(_, ~entityMod, ~inMemoryStore)
+  })
 
-  {{/each}}
   //In the event of a rollback, rollback all meta tables based on the given
   //valid event identifier, where all rows created after this eventIdentifier should
   //be deleted
   let rollbackTables = switch inMemoryStore.rollBackEventIdentifier {
-  | Some(eventIdentifier) =>
-    [
+  | Some(eventIdentifier) => [
       DbFunctions.EntityHistory.deleteAllEntityHistoryAfterEventIdentifier(
+        _,
         ~isUnorderedMultichainMode=config.isUnorderedMultichainMode,
+        ~eventIdentifier,
       ),
-      DbFunctions.RawEvents.deleteAllRawEventsAfterEventIdentifier,
-      DbFunctions.DynamicContractRegistry.deleteAllDynamicContractRegistrationsAfterEventIdentifier,
```

Comment:
Raw events does not delete anymore, it just allows duplicate entries on rollback. (For now)

Dynamic contracts is handled in entity history